lib/crypto.js

```diff
@@ -647,9 +647,21 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
-exports.randomBytes = exports.pseudoRandomBytes = randomBytes;
-
-exports.rng = exports.prng = randomBytes;
+exports.randomBytes = util._deprecateSync(randomBytes,
+    '`crypto.randomBytes(size)` is deprecated. ' +
+    'Please use `crypto.randomBytesSync(size)` instead.');
+exports.randomBytesSync = randomBytes;
+exports.pseudoRandomBytes = util.deprecate(randomBytes,
+    '`crypto.pseudoRandomBytes()` is deprecated. ' +
+    'Please use `crypto.randomBytes()` instead.');
```

Comment:
@jonathanong We are not trying to find fault in your code, but we are trying to make sure that the code is maintainable. To get this particular deprecation messages consistent, me and @silverwind worked for over a month [here](https://github.com/nodejs/io.js/pull/1892). So, if you understand the common stuff like this next time onwards the review process will be shorter :-) Hope you would contribute more in the future :-)